Stage1/environment/env_1d.py

```diff
@@ -20,7 +50,7 @@ def __init__(self, plan_file_path: str):
             0: speed, px/sec
```

Comment:
What does "1d" in the name Env1d mean? I suggest naming this class as "Environment"

Reply:
This comment is not for this PR. 1d was there before. Agree with naming. Please create issue for refactor.